src/app/profile/profile/profile-species.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="(data$ | async) !== null" class="profile-species">
+  <div class="profile-species-component">
+    <a [routerLink]="['/profile', (data$ | async).id]" class="profile-species-component__title">
+      <h2>{{ 'Zurück' | translate }}</h2>
```

Comment:
Content and style are not matching the design. If we omit the arrow for now we should mention it in #355.

Reply:
I'm assuming that the arrow is part of Block 1 and should be done there?

Reply:
added to #355